LPS-27113 Regen

epgarcia opened the pull request.

jkappler committed:
LPS-27113 Move logic from jsp file to java class

jkappler committed:
LPS-27113 Create necessary methods in NavItem class in preparation for ADT

jkappler committed:
LPS-27113 Change Layout in favour of NavItem

jkappler committed:
LPS-27113 Create PortletDisplayTemplateHandler class for the module

jkappler committed:
LPS-27113 Establish display templates location for the module and rights to add new ones

jkappler committed:
LPS-27113 Add DDM attributes in navigation tag

jkappler committed:
LPS-27113 Add new configuration attribute in preparation for ADT

jkappler committed:
LPS-27113 Apply ADT in navigation menu portlet

jkappler committed:
LPS-27113 Add necessary variables in Display Template Handler

jkappler committed:
LPS-27113 Create portlet display templates based on current display templates

jkappler committed:
LPS-27113 Add default ddmTemplateKey to the modules configuration

jkappler committed:
LPS-27113 Create View Portlet Provider for the module

jkappler committed:
LPS-27113 Create macro for navigation menu

jkappler committed:
LPS-27113 Make configuration values available in ADT palette

jkappler committed:
LPS-27113 Remove methods since everything was moved to ADT

jkappler committed:
LPS-27113 Add language keys for new templates

jkappler committed:
LPS-27113 Language key for ADT Palette

jkappler committed:
LPS-27113 Use new tag for template renderer

Commit:
LPS-27113 SF

Commit:
LPS-27113 Add macro template type

Commit:
LPS-27113 Extract common logic to macro

Commit:
LPS-27113 Rename

jkappler committed:
LPS-27113 Change rootLayoutLevel to Integer in palette and convert to number for comparisson

Commit:
LPS-27113 Fix tests

Commit:
LPS-56794 Inline for now for taglibs that will remain in core.

brianchandotcom committed:
LPS-27113 Forgot clean up, and sort

brianchandotcom committed:
LPS-27113 Rename

brianchandotcom committed:
LPS-27113 Simplify

brianchandotcom committed:
LPS-27113 sort by usage

brianchandotcom committed:
LPS-27113 match ClusterNode#equals

brianchandotcom committed:
LPS-27113 not used? Please revert this commit... it's still referenced in the TLD

Commit:
LPS-27113 Prevent ORA-01461 error

Commit:
LPS-27113 Remove old navigation styling mechanism

Commit:
LPS-27113 Regen

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 45 minutes 8 seconds</p><h4>Job Results:</h4><p>12 Jobs Passed.<br />2 Jobs Failed.</p><h1>See the following links for more information.</h1><ul><li><h4><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest(master)/798/">test-portal-acceptance-pullrequest(master)</a></h4><pre>Completed with the status of FAILURE.</pre></li><li><h4><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2652/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat-mysql/portal-smoke</a></h4><h5>Job Results:</h5><p>1 Test Passed.<br />1 Test Failed.</p><ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PortalSmoke%23Smoke,label_exp=!master/2652//testReport/com.liferay.poshi.runner/PoshiRunner/test_PortalSmoke_Smoke_">PoshiRunner.test[PortalSmoke#Smoke]</a> - <strong>brianchandotcom</strong> - <a href="https://github.com/brianchandotcom/liferay-portal/pull/28789">PR#28789</a> - <a href="https://test-1-4.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/798/jenkins-report.html">Jenkins Report</a> - <a href="https://test-1-4.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/798/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PortalSmoke%23Smoke,label_exp=!master/functional-tomcat-mysql/portal-smoke/index.html">Poshi Report</a> - <a href="https://test-1-4.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest%28master%29/builds/798/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PortalSmoke%23Smoke,label_exp=!master/functional-tomcat-mysql/portal-smoke/summary.html">Poshi Summary</a> - <a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=PortalSmoke%23Smoke,label_exp=!master/2652//console">Console Output</a></li></ul></li><li><h4><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2661/">test-portal-acceptance-pullrequest-batch(master)/integration-db2</a></h4></li><li><h4><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2653/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql</a></h4></li><li><h4><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2657/">test-portal-acceptance-pullrequest-batch(master)/integration-oracle</a></h4></li><li><h4><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2665/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql</a></h4></li><li><h4><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2698/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2</a></h4><h5>Job Results:</h5><p>3089 Tests Passed.<br />1 Test Failed.</p><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=1,label_exp=!master/2698//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXmlLog">PortalLogAssertorTest.testScanXmlLog</a></li></ul></li><li><h4><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2693/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql</a></h4></li><li><h4><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2631/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle</a></h4></li><li><h4><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2653/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql</a></h4></li><li><h4><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2662/">test-portal-acceptance-pullrequest-batch(master)/modules-unit</a></h4><h5>Job Results:</h5><p>958 Tests Passed.<br />1 Test Failed.</p><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=1,label_exp=!master/2662//testReport/com.liferay.portal.tools.sample.sql.builder/SampleSQLBuilderTest/testGenerateAndInsertSampleSQL">SampleSQLBuilderTest.testGenerateAndInsertSampleSQL</a></li></ul></li><li><h4><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2654/">test-portal-acceptance-pullrequest-batch(master)/plugins</a></h4></li><li><h4><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2658/">test-portal-acceptance-pullrequest-batch(master)/portal-web</a></h4></li><li><h4><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2666/">test-portal-acceptance-pullrequest-batch(master)/tck</a></h4></li><li><h4><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2699/">test-portal-acceptance-pullrequest-batch(master)/unit</a></h4></li><li><h4><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/775/">test-portal-acceptance-pullrequest-source(master)</a></h4></li></ul>

brianchandotcom commented:
@epgarcia 

Can you resend?

1.) The text for display.templates.config in portlet.properties mentions the Breadcrumb portlet, is that acucrate?

2.) I see Language keys copied but I don't see any removed. We should avoid, at all possible, duplicate Language keys like that. If it's not moved, they should be moved to a *-lang module that other modules then share.

Thx @epgarcia

epgarcia commented:
Hi @brianchandotcom 

1.) You're right, sorry, it should be "Navigation Menu". I fixed it in my new pr (see brianchandotcom/liferay-portal/pull/28802).

2.) We didn't copy any language keys, but reused the existing translations of the old pre-defined styles for the new ADT names. Then we added new keys for any non-existing translation. I've double checked that we don't have any duplicated keys, but if you find any please tell me and I'll fix it.

Thanks!
